app/src/main/java/com/sun/dailynote/data/database/Database.java

```diff
@@ -0,0 +1,112 @@
+package com.sun.dailynote.data.database;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.SQLException;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import com.sun.dailynote.data.model.Note;
+
+import java.text.SimpleDateFormat;
+
+public class Database extends SQLiteOpenHelper {
+    public static final String NOTE_TABLE_NAME = "Note";
+    private static final String NOTE_ID = "id";
+    private static final String NOTE_NAME = "name";
+    private static final String NOTE_CONTENT = "content";
+    private static final String NOTE_STATUS = "status ";
+    private static final String NOTE_CREATE_DAY = "createDay";
```

Comment:
```suggestion
    private static final String NOTE_CREATE_DAY = "create_day";
```
tên bảng nên đặt tên là số nhiều, tên collumn đặt theo kiểu under_score